cookbooks/aws-parallelcluster-scheduler-plugin/recipes/install_python.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster-scheduler-plugin
+# Recipe:: install_python
+#
+# Copyright 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+#
+install_pyenv node['cluster']['python-version'] do
+  prefix node['cluster']['scheduler_plugin']['pyenv_root']
```

Comment:
cannot we reuse the pyenv installed in `prefix node['cluster']['system_pyenv_root']`?